Homework2.Tests/ArrayExtensionsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Linq;
+using Xunit;
+
+namespace Homework2.Tests
+{
+    public class ArrayExtensionsTests
+    {
+        [Theory]
+        [InlineData(999, 4)]
+        [InlineData(999, 0)]
+        [InlineData(999, 10)]
+        public void AddAt_adds_at_idx(int item, int idx)
```

Comment:
What happens if idx is higher than array size?

Reply:
What about negative?

Reply:
same for all methods here.

Reply:
Changed existing and added new tests. Also added exceptions to the methods (and changed the `RemoveAt(int idx)` one).